packages/core/src/anchor/ethereum/ethereum-anchor-validator.ts

```diff
@@ -6,6 +6,11 @@ import { AnchorProof, AnchorValidator, DiagnosticsLogger } from '@ceramicnetwork
 import { Block, TransactionResponse } from '@ethersproject/providers'
 import { base16 } from 'multiformats/bases/base16'
```

Comment:
I don't think we need the `base16` dependency here as we already import and use uint8arrays.